apps/web/src/components/SearchRequestTable/SearchRequestsTable.tsx

```diff
@@ -0,0 +1,320 @@
+    () => [
+      {
+        label: intl.formatMessage({
+          defaultMessage: "Action",
+          id: "TDTE1c",
+          description:
+            "Title displayed for the search request table edit column.",
+        }),
+        id: "action",
+        accessor: (d) =>
+          tableViewItemButtonAccessor(
+            paths.searchRequestView(d.id),
+            intl.formatMessage({
+              defaultMessage: "request",
+              id: "gLtTaW",
+              description:
+                "Text displayed after View text for Search Request table view action",
+            }),
+            d.fullName || "",
+          ),
+      },
+      {
+        label: intl.formatMessage({
+          defaultMessage: "ID",
+          id: "f24Z8p",
+          description: "Title displayed on the search request table id column.",
+        }),
+        id: "id",
+        sortColumnName: "id",
+        accessor: (d) => d.id,
+      },
+      {
+        label: intl.formatMessage({
+          defaultMessage: "Date Received",
+          id: "r2gD/4",
+          description:
+            "Title displayed on the search request table requested date column.",
+        }),
+        id: "requestedDate",
+        sortColumnName: "created_at",
+        accessor: (d) => dateCell(d.requestedDate, intl),
+      },
+      {
+        label: intl.formatMessage({
+          defaultMessage: "Status",
+          id: "t3sEc+",
+          description:
+            "Title displayed on the search request table status column.",
+        }),
+        id: "status",
+        sortColumnName: "done_at",
+        accessor: (d) => statusAccessor(d.status, intl),
+      },
+      {
+        label: intl.formatMessage({
+          defaultMessage: "Manager",
+          id: "7FtbwK",
+          description:
+            "Title displayed on the search request table manager column.",
+        }),
+        id: "manager",
+        sortColumnName: "full_name",
+        accessor: (d) => d.fullName,
+      },
+      {
+        label: intl.formatMessage({
+          defaultMessage: "Email",
+          id: "hiZAeF",
+          description:
+            "Title displayed on the search request table email column.",
+        }),
+        id: "email",
+        sortColumnName: "email",
+        accessor: (d) => d.email,
+      },
+      {
+        label: intl.formatMessage({
+          defaultMessage: "Department",
+          id: "i3C5Hn",
+          description:
+            "Title displayed on the search request table department column.",
+        }),
+        id: "department",
+        accessor: (d) => getLocalizedName(d.department?.name, intl),
+      },
+      {
+        label: intl.formatMessage({
+          defaultMessage: "Job Title",
+          id: "8hee5d",
+          description:
+            "Title displayed on the search request table job title column.",
+        }),
+        id: "jobTitle",
+        sortColumnName: "job_title",
+        accessor: (d) => d.jobTitle,
```

Comment:
I think this needs to be localized, too.
![image](https://github.com/GCTC-NTGC/gc-digital-talent/assets/8978655/65840469-8757-45a3-a261-3f6381d079d8)

Reply:
`jobTitle` is not an object with en and fr keys

![image](https://github.com/GCTC-NTGC/gc-digital-talent/assets/40485260/6c6aeedf-1bc2-4692-9d39-3461f30ca229)

Reply:
Oh, you're right - this is a non-localized free-form field.